### PR TITLE
fix(tier4_perception_launch): add object_merger of far_objects to fusion for Camera-LiDAR-Radar fusion

### PR DESCRIPTION
**Pull request by scepter914**
## Description

This PR add `object_merger` of far_objects to fusion for Camera-LiDAR-Radar fusion and refactor according to it.

- Add object_merger of far_objects in launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

Until the tracking merger can operate stably, this PR temporarily added near_object and far_object mergers to the detection side and connected the far detection pipeline to tracking.
`near_object` come from existing Camera-LiDAR-Radar fusion and `far_object` come from radar detection.

- Add radar lanelets filter parameter to detection launch

I added radar lanelets filter parameter to detection launch to use config parameter from autoware.launch

- Refactor parameter name and comment

To make it easier to understand parameters, I renamed parameter names and comment.

## Related links
* https://github.com/autowarefoundation/autoware_launch/pull/566

## Tests performed

Tests by rosbag.

Screenshot from 2023-09-14 17-20-34

Red objects are detection/radar/far_object.
Other objects are tracking/objects.
Grid size is 10m.
I could confirm that the long-distance detection was working properly with tracking.

## Interface changes

None

## Effects on system behavior

As adding faraway detection, dynamic objects at a distance become visible, so it effects to planning behavior.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [x] The PR has been properly tested.
- [x] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_perception_launch/launch/perception.launch.xml [hunk 1]
```diff
@@ -88,15 +90,15 @@
     description="options: `traffic_light_classifier_mobilenetv2_batch_*` or `traffic_light_classifier_efficientNet_b1_batch_*`. The batch number must be either one of 1, 4, 6"
   />
 
-  <!-- Camera Lidar Fusion Parameters -->
+  <!-- Camera-Lidar Fusion Parameters -->
```
Comment (yukkysaito) [inline launch/tier4_perception_launch/launch/perception.launch.xml:93]: How about the following to be consistent with the others?
```suggestion
  <!-- Camera-Lidar fusion param -->
```

Comment (scepter914) [reply launch/tier4_perception_launch/launch/perception.launch.xml]: I also fixed at [23cb2f1](https://github.com/autowarefoundation/autoware.universe/pull/4995/commits/23cb2f1119f70e5ac0dc172749e9b1ee7907944e)

**Review by yukkysaito — APPROVED** [review]
LGTM

File: launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-  <!-- lidar param -->
+  <!-- LiDAR parameters -->
   <arg name="input/pointcloud"/>
   <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`, `clustering`"/>
   <arg name="use_pointcloud_map" default="true" description="use pointcloud map in detection"/>
   <arg name="use_object_filter" default="true" description="use object filter"/>
   <arg name="use_pointcloud_container" default="false" description="use pointcloud container for detection preprocessor"/>
   <arg name="container_name" default="pointcloud_container"/>
-  <arg name="score_threshold" default="0.20"/>
+  <arg name="lidar_detection_score_threshold" default="0.35"/>
```
Comment (miursh) [inline launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml:10]: [Q] Is this change in threshold intended?

File: launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
 
-  <!-- radar param -->
+  <!-- Radar parameters -->
   <arg name="input/radar" default="/sensing/radar/detected_objects"/>
+  <arg name="radar_lanelet_filtering_range_param" default="$(find-pkg-share detected_object_validation)/config/object_position_filter.param.yaml"/>
```
Comment (miursh) [inline launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml:14]: Should be this?
```suggestion
  <arg name="radar_lanelet_filtering_range_param" default="$(find-pkg-share detected_object_validation)/config/object_lanelet_filter.param.yaml"/>
```

File: launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml (hunk 2 above)
Comment (scepter914) [reply launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml:10]: Yes, I maintenance LiDAR-Radar fusion and set same parameters for Camera-LiDAR-Radar fusion.

File: launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml (hunk 3 above)
Comment (scepter914) [reply launch/tier4_perception_launch/launch/object_recognition/detection/lidar_radar_based_detection.launch.xml]: Fixed other default parameters at [fac721b](https://github.com/autowarefoundation/autoware.universe/pull/4995/commits/fac721b4051f4d1a2f7f659905b4204a4a048b48)

**Review by miursh — APPROVED** [review]
LGTM

**Comment by scepter914** [comment]
When merge from main branch, conflicts has occurred because new PR is merged to main branch.
So I fixed it, but I could not fix DCO problem.
I'm sorry that I will make PR again.

**Pull request closed without merging**